src/sentry/migrations/0840_add_explore_saved_query_model.py

```diff
@@ -0,0 +1,108 @@
+# Generated by Django 5.1.5 on 2025-03-07 02:12
+
+import django.db.models.deletion
+import django.utils.timezone
+from django.db import migrations, models
+
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.foreignkey
+import sentry.db.models.fields.hybrid_cloud_foreign_key
+import sentry.db.models.fields.jsonfield
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production.
+    # This should only be used for operations where it's safe to run the migration after your
+    # code has deployed. So this should not be used for most operations that alter the schema
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = False
+
+    dependencies = [
+        ("sentry", "0839_add_visibility_column_to_groupsearchview"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="ExploreSavedQuery",
+            fields=[
+                (
+                    "id",
+                    sentry.db.models.fields.bounded.BoundedBigAutoField(
+                        primary_key=True, serialize=False
+                    ),
+                ),
+                (
+                    "created_by_id",
+                    sentry.db.models.fields.hybrid_cloud_foreign_key.HybridCloudForeignKey(
+                        "sentry.User", db_index=True, null=True, on_delete="SET_NULL"
+                    ),
+                ),
+                ("name", models.CharField(max_length=255)),
+                ("query", sentry.db.models.fields.jsonfield.JSONField(default=dict)),
+                ("date_created", models.DateTimeField(auto_now_add=True)),
+                ("date_updated", models.DateTimeField(auto_now=True)),
+                (
+                    "visits",
+                    sentry.db.models.fields.bounded.BoundedBigIntegerField(default=1, null=True),
+                ),
+                (
+                    "last_visited",
+                    models.DateTimeField(default=django.utils.timezone.now, null=True),
+                ),
+                ("dataset", sentry.db.models.fields.bounded.BoundedPositiveIntegerField(default=0)),
+                ("is_multi_query", models.BooleanField(blank=True, null=True)),
```

Comment:
Do you need null on this column? What does null mean that true/false doesn't?

Reply:
Good point, I don't need a third state so I've removed blank and null, and defaulted to false.